apps/governance/src/routes/proposals/propose/freeform/propose-freeform.tsx

```diff
@@ -165,7 +164,6 @@ export const ProposeFreeform = () => {
                 voteMaxClose={params.governance_proposal_freeform_maxClose}
               />
 
-              <ProposalFormSubmit isSubmitting={isSubmitting} />
```

Comment:
Is this just to prevent submission? if so you you might want to prevent submission on enter press as the form as an onSubmit handler